### PR TITLE
HUE-8789: [jobbrowser] Can't kill job in mini job browser

### PR DESCRIPTION
**Pull request by MishaKysliuk**

### CODE REVIEW

**Comment by MishaKysliuk** [comment]
https://issues.cloudera.org/browse/HUE-8789
https://review.cloudera.org/r/13797/

File: apps/jobbrowser/src/jobbrowser/templates/job_browser.mako [hunk 1]
```diff
@@ -2614,6 +2614,10 @@ ${ commonheader("Job Browser", "jobbrowser", user, request) | n,unicode }
                 var rerunModalData = $(this).data('modal');
                 rerunModalData.$backdrop.appendTo("#jobbrowserMiniComponents");
             });
+            $('#killModal${ SUFFIX }').on('shown', function (e) {
+                 var rerunModalData = $(this).data('modal');
```
Comment (romainr) [inline apps/jobbrowser/src/jobbrowser/templates/job_browser.mako]: nit: rerunModalData --> killModalData or modalData?

Comment (MishaKysliuk) [reply apps/jobbrowser/src/jobbrowser/templates/job_browser.mako]: Fixed

**Comment by romainr** [comment]
Thanks!

**Comment by zhonglaowu** [comment]
Hi，this commit not work on hue 4.3， more modification need to do？

**Comment by zhonglaowu** [comment]
/hue/jobbrowser#!jobs

**Comment by zhonglaowu** [comment]
When miniJobbrowser was first open in jobbrowser, it was work,
image
But when miniJobbrowser  first open in  editor site, it still not work and can't kill.
image

**Pull request merged**